tests/test_image_upload.py

```diff
@@ -74,6 +89,20 @@ def windows_path(path: str):
         yield path
 
 
+# Whenever you modify expected_pass_list, delete files with names from the old list
+# from test_data, set autouse to True and run test_image_upload_file to regenerate files
+@pytest.fixture(autouse=False, scope="module")
+def create_test_files(test_data_path, change_test_dir_module):
```

Comment:
Is this even used anywhere?

Reply:
Updated the comment above to be clearer that this is not intended to be used normally. 

The fixture was created to initialize the files before every test run and delete them afterwards, but the way parallelization works threw a spanner into that plan. It was easier, in the end, to just create the files once and commit them, and I used this fixture to create them. 

We can also just remove this part, and leave it to those modifying tests to create the files however they want.